src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -26,6 +38,29 @@ public class CommandPat
         private const string OutputOption = "--output";
         private const string OutputHelp = "How PAT information is displayed. [default: token]\n[possible values: none, status, token, base64, header, headervalue, json]";
 
+        private const string DomainOption = "--domain";
+        private const string DomainHelp = "The preferred domain used when acquiring Azure Active Directory access tokens. [default: microsoft.com]";
+
+        private const string AccessTokenPrompt = "AzureAuth ADO PAT";
+        private static readonly IEnumerable<AuthMode> AccessTokenAuthModes = new[] { AuthMode.Default };
+        private static readonly TimeSpan AccessTokenTimeout = TimeSpan.FromMinutes(15);
+
+        private static readonly string LockfilePath = Path.Combine(Path.GetTempPath(), AzureAuth.Ado.Constants.PatLockfileName);
+
+        // On all platforms the PAT cache should be in the same directory as a typical AzureAuth installation.
+        //   - On Windows this is usually `%LOCALAPPDATA%\Programs\AzureAuth`.
+        //   - On Unix-like platforms this is usually `~/.azureauth`.
+#if PlatformWindows

```

Comment:
I'm curious why this is not generating the same warning that Rick sees in program.cs (or maybe it is and we're just not seeing it here) - about this being reachable on all platforms.

Reply:
No idea. Maybe we have a StyleCop configuration issue? 🤔